plugin/src/main/java/io/fabric8/maven/plugin/PushMojo.java

```diff
@@ -95,8 +100,8 @@ protected String getLogPrefix() {
                 generator != null ?
                     generator :
                     ProfileUtil.extractProcesssorConfiguration(ProfileUtil.GENERATOR_CONFIG, profile, resourceDir);
-            return GeneratorManager.generate(configs, generatorConcfig, project, log, mode, buildStrategy, false);
-        } catch (IOException e) {
+            return GeneratorManager.generate(configs, generatorConcfig, project, session, goalFinder, log, mode, buildStrategy, false);
+        } catch (Exception e) {
```

Comment:
Here we could leave the IOException if customize() would throw the IOException as described above.

Alternatively I would catch `IOException | MavenExecutionException`, not plain `Exception`.